docs/team.rst

```diff
@@ -1,43 +1,54 @@
 Team
 ====
 
-Write the Docs is a community effort that is enabled by a wonderful team of folks.
-We are always looking for more people to help organize conferences,
-meetups,
-or take a leadership role in the community.
-This page documents the status of different members of the community,
-and also explains some of the roles that people are filling.
+The different facets of the Write the Docs community is kept running smoothly by
```

Comment:
s/community is kept running smoothly/community are kept running smoothly/

my bad